MarsOnboarding24/StepDefinitions/LanguageStepDefinitions.cs

```diff
@@ -0,0 +1,157 @@
+using MarsOnboarding24.Pages;
+using MarsOnboarding24.Utilities;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using System;
+using TechTalk.SpecFlow;
+
+namespace MarsOnboarding24.StepDefinitions
+{
+    [Binding]
+    public class LanguageStepDefinitions : CommonDriver
+    {
+        LoginPage loginPageObj = new LoginPage();
+        LanguagePage languagePageObj = new LanguagePage();
+
+        [AfterScenario]
+
+        public void QuitTestRun()
+        {
+            languagePageObj.ResetTest(driver);
```

Comment:
please do not pass driver in action methods in pages! Instead create a common driver class and extend pages to common driver

Reply:
https://codebun.com/how-to-implement-hooks-in-specflow-c/